clients/java/src/test/java/io/camunda/zeebe/client/impl/worker/JobWorkerImplTest.java

```diff
@@ -121,6 +121,34 @@ public void shouldBackoffWhenGatewayRespondsWithResourceExhausted() {
                 assertThat(gateway.getTimeBetweenLatestPolls()).isGreaterThan(SLOW_POLL_THRESHOLD));
   }
 
+  @Test
+  public void shouldBackoffWhenStreamEnabledAndOnPollSuccess() {
```

Comment:
:x: Should be that we back off on success if the response is empty, i.e. if the gateway was not responding with any jobs